tests/ui/parser/raw/raw-idents.rs

```diff
@@ -0,0 +1,158 @@
+//@ check-pass
+//@ revisions:e2015 e2018 e2021 e2024
+//[e2015]@ edition:2015
+//[e2018]@ edition:2018
+//[e2021]@ edition:2021
+//[e2024]@ edition:2024
```

Comment:
Sorry, I think this needs to be:

```suggestion
//@[e2015] edition:2015
//@[e2018] edition:2018
//@[e2021] edition:2021
//@[e2024] edition:2024
```

Reply:
https://github.com/rust-lang/rust/issues/137146